images-list

```diff
@@ -1,6 +1,19 @@
-# This list should be sorted alphabetically. Do not change existing entries. Comments lines may be started with # or //.
+# This list should be sorted lexographically. Do not change existing entries. Comments lines may be started with # or //.
 # This list was reset in order to prevent breakage when moving to multiarch mirroring. See the following PR for more information:
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+coredns/coredns rancher/coredns-coredns 1.7.0
+coredns/coredns rancher/coredns-coredns 1.7.1
+istio/coredns-plugin rancher/istio-coredns-plugin 0.2-istio-1.1
```

Comment:
Was thinking we got rid of coredns in 1.7, but maybe not. This looks to be only existing image in the PR.

Reply:
From their annoucements, it seemed like they were going to phase out all `addonComponents` but coredns is not part of their [integrations](https://istio.io/latest/docs/ops/integrations/) list. The current demo profile for 1.7.1 still contains `istiocoredns` as an `addonComponents`